### ISSUE TITLE
Deprecate or remove t.conch.ssh.keys.signData, verifySignature and printKey

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4105]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#4105|
|Type|task|
|Created|2009-11-13 20:25:50Z|

```signData```, ```verifySignature``` and ```printKey``` in [twisted/conch/ssh/keys.py](twisted/conch/ssh/keys.py) have been deprecated since Conch 0.9, so let's remove those.

<details><summary>Searchable metadata</summary>

```
trac-id__4105 4105
type__task task
reporter__thijs thijs
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__ 
time__1258143950000000 1258143950000000
changetime__1275243193000000 1275243193000000
version__None None
owner__cyli cyli
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_1"></a>|@cyli commented|
|-|-|

Deprecated 11/08/2007

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_2"></a>|@cyli commented|
|-|-|

(In [29073]) Removed twisted.conch.ssh.keys.signData, twisted.conch.ssh.keys.verifySignature,
and twisted.conch.ssh.keys.printKey, which have been deprecated since 2007.
 
refs [#4105](https://github.com/twisted/twisted/issues/4105)

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_3"></a>|@cyli removed owner|
|-|-|

Also removed the useless tests from twisted.conch.test.test_keys.

As a lot of the conch deprecation removals are related (they all affect twisted.conch.test.test_keys), I put the fixes in the same branch.  This branch also attempts to fix:  [#4097](https://github.com/twisted/twisted/issues/4097), [#4098](https://github.com/twisted/twisted/issues/4098), [#4099](https://github.com/twisted/twisted/issues/4099), [#4100](https://github.com/twisted/twisted/issues/4100), [#4101](https://github.com/twisted/twisted/issues/4101), [#4102](https://github.com/twisted/twisted/issues/4102), [#4103](https://github.com/twisted/twisted/issues/4103), [#4104](https://github.com/twisted/twisted/issues/4104).  (Each one was a separate commit.)

Build results:  http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/remove-deprecated-conch-methods-4097

**Comment by twisted-trac** [comment]
|[<img alt="mithrandi's avatar" src="https://avatars.githubusercontent.com/u/856430?s=50" width="50" height="50">](https://github.com/mithrandi)<a name="note_4"></a>|@mithrandi set owner to @cyli|
|-|-|

Looks good, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_5"></a>|@cyli set status to `closed`|
|-|-|

(In [29094]) Merge remove-deprecated-conch-methods-4097: removed a bunch of deprecated methods from twisted.conch

Author: cyli 
Reviewer: mithrandi
Fixes: [#4097](https://github.com/twisted/twisted/issues/4097)
Fixes: [#4098](https://github.com/twisted/twisted/issues/4098)
Fixes: [#4099](https://github.com/twisted/twisted/issues/4099)
Fixes: [#4100](https://github.com/twisted/twisted/issues/4100)
Fixes: [#4101](https://github.com/twisted/twisted/issues/4101)
Fixes: [#4102](https://github.com/twisted/twisted/issues/4102)
Fixes: [#4103](https://github.com/twisted/twisted/issues/4103)
Fixes: [#4104](https://github.com/twisted/twisted/issues/4104)
Fixes: [#4105](https://github.com/twisted/twisted/issues/4105)

Removes twisted.conch.ssh.asn1, which has been deprecated since Twisted 9.0.  Removes twisted.conch.ssh.common.Entropy, since Entropy.get_bytes has been deprecated since 2007 and is the only attribute of Entropy.  Removed the following methods in twisted.conch.ssh.keys, which have been deprecated since Conch 0.9 (2007): getPublicKeyString, makePublicKeyString, getPublicKeyObject, getPrivateKeyObject, makePrivateKeyString, makePublicKeyBlob, signData, verifySignature, and printKey.  Also removed the relevant tests, and updated the conch documentation (primarily in the sample client and server).
